builtin/aws/alb/plugin.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.23.0
-// 	protoc        v3.11.2
+// 	protoc-gen-go v1.25.0
+// 	protoc        (unknown)
```

Comment:
Why is the version `unknown` here?

Reply:
Good question. I have protoc installed on Linux, but can't seem to find out why my generated version always ends up `(unknown)` 🤔